packages/@aws-cdk/aws-ivs-alpha/lib/channel.ts

```diff
@@ -162,11 +200,16 @@ export class Channel extends ChannelBase {
       throw new Error(`channelName must contain only numbers, letters, hyphens and underscores, got: '${this.physicalName}'`);
     }
 
+    if ((props.type === ChannelType.STANDARD || props.type === ChannelType.BASIC) && props.preset !== undefined) {
+      throw new Error('preset cannot be used when STANDARD or BASIC channel type');
+    }
+
     const resource = new CfnChannel(this, 'Resource', {
       authorized: props.authorized,
       latencyMode: props.latencyMode,
       name: this.physicalName,
       type: props.type,
+      preset: props.preset,
```

Comment:
Taking into account the revisions after the review, I changed it to set props.preset directly.
This is because if ChannelType is ADVANCED_SD or ADVANCED_HD, even if preset becomes undefined, Cfn will set the default value.

Reply:
Once again, you can use the shorthand:
```suggestion
      preset,
```

Reply:
Thank you.
I had a misunderstanding about the shorthand.

Reply:
@nmussy 
No problem. I reverted it.